### ISSUE TITLE
document pymatgen methods?

### ISSUE THREAD

**Issue by computron** [materialsproject/mapidoc#13]
Technically pymatgen and mapi are separate, but perhaps it would be useful to document some of the pymatgen built-ins here. See:

https://groups.google.com/forum/#!topic/pymatgen/d3q1q6TgDTY

Perhaps in a different folder than "materials" - e.g. pymatgen?

**Comment by shyuep** [comment]
I would suggest we do the reverse. For each PMG method, we put a link to the relevant mapi link. E.g., `get_structure_by_id` basically points to the `https://github.com/materialsproject/mapidoc/tree/master/materials/structure` in the doc. 

I can also add a link to pymatgen's built in methods API `http://pymatgen.org/pymatgen.ext.matproj.html?highlight=mprester`

**Comment by computron** [comment]
If I understand correctly, you want to put the documentation in pymatgen instead of mapidoc.

I certainly support linking pymatgen to mapidoc, but currently mapidoc is "missing" information. e.g., for example, it's possible to get phonon information (e.g. through pymatgen) but this information is not present anywhere in mapidoc. Since mapidoc is supposed to be the ultimate reference of things you can get with the Materials API, it should be documented somewhere in mapidoc itself that phonon information is available.

Of course, ideally there would be some kind of phonon key in mapidoc that pymatgen can point to. But currently, there is no such key

**Comment by shyamd** [comment]
Sounds like another thing to add the dissemination checklist.

Shyam Dwaraknath
Materials Research Scientist
Lawrence Berkeley Labs

From: Anubhav Jain <notifications@github.com> <notifications@github.com>
Reply: materialsproject/mapidoc
<reply@reply.github.com>
<reply@reply.github.com>
Date: August 1, 2018 at 10:00:50 AM
To: materialsproject/mapidoc <mapidoc@noreply.github.com>
<mapidoc@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
<subscribed@noreply.github.com>
Subject:  Re: [materialsproject/mapidoc] document pymatgen methods? (#13)

If I understand correctly, you want to put the documentation in pymatgen
instead of mapidoc.

I certainly support linking pymatgen to mapidoc, but currently mapidoc is
"missing" information. e.g., for example, it's possible to get phonon
information (e.g. through pymatgen) but this information is not present
anywhere in mapidoc. Since mapidoc is supposed to be the ultimate reference
of things you can get with the Materials API, it should be documented
somewhere in mapidoc itself that phonon information is available.

Of course, ideally there would be some kind of phonon key in mapidoc that
pymatgen can point to. But currently, there is no such key

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/materialsproject/mapidoc/issues/13#issuecomment-409646937>,
or mute the thread

**Comment by shyuep** [comment]
@computron Yes, I know that. There is actually an automated code to generate the full mapidoc from a MongoDB doc (which someone in the MP team needs to provide since I am retired from day-to-day querying of the db...). I would suggest we use something that has a lot of properties, e.g., Al2O3 that @kristinpersson  mentioned previously. And yes, this should be added to the dissemination checklist. @richardtran415 - pls follow up.
